### PR TITLE
Update documents for rhosp13

### PR DESCRIPTION
**Pull request by yong-huang**
Major Changes:
- Update container image pulling process
- Add instruction of IPv6 related configurations

### CODE REVIEW

File: cinder/README.md [hunk 1]
```diff
@@ -25,15 +25,18 @@ The Dell EMC Cinder container image contains following RPM packages:
 
 The formal Dell EMC container image is published to [Red Hat Container Catalog](https://access.redhat.com/containers/)
 
+Red Hat OpenStack Platform supports remote registry and local registry for overcloud deployment. In this document, we only introduce local registry.
```
Comment (amagicsun) [inline cinder/README.md]: we only introduce local registry?

File: manila/README.md [hunk 2]
```diff
@@ -95,15 +114,18 @@ emc_ssl_cert_verify=False
 
 The formal Dell EMC container image is published to [Red Hat Container Catalog](https://access.redhat.com/containers/)
 
+Red Hat OpenStack Platform supports remote registry and local registry for overcloud deployment. In this document, we only introduce local registry.
```
Comment (amagicsun) [inline manila/README.md]: Ditto

File: cinder/README.md (hunk 1 above)
Comment (yong-huang) [reply cinder/README.md]: Yes, we only use local registry in our deployment.
There are very detail instructions for remote registry in Red Hat document, so we don't need to cover it again.

Comment (amagicsun) [reply cinder/README.md]: Should we mention this to the user?

Comment (yong-huang) [reply cinder/README.md]: We assume that they have enough knowledge to deploy osp, otherwise there are lots of things we need to mention in this document.

**Pull request merged**